common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala

```diff
@@ -87,6 +89,10 @@ object Block {
     val displayedDateTimes = ArticleDateTimes.makeDisplayedDateTimesDCR(dateTimes, request)
     val campaigns = page.getJavascriptConfig.get("campaigns")
 
+    val contributors = block.contributors flatMap { contributorId =>
+      tags.find(_.id == s"profile/$contributorId").map(tag => Contributor(tag.title, tag.bylineImageUrl))
+    }
```

Comment:
Could/should we use `tag.type === "Contributor"` here? I can't really see a use case and I admit its debatable if the extra level of checking is beneficial or not so I will leave this up to you

Reply:
I guess it wouldn't hurt to add an extra check?
Elsewhere we seem to count on the id format to work out if a tag is of type contributor:
https://github.com/guardian/frontend/blob/main/common/app/model/Tag.scala#L192

I can't think of any strong arguments one way or the other..